crowdsec-docs/unversioned/getting_started/installation/linux.mdx

```diff
@@ -0,0 +1,146 @@
+---
+id: linux
+title: Linux
+pagination_prev: getting_started/pre_requisites
+pagination_next: getting_started/next_steps
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+# Linux
```

Comment:
Page title should be even more explicit ie. "Installation of the CrowdCec Security Engine" or something ?